translations/tournament_mode.tex/pl.po

```diff
@@ -209,3 +271,30 @@ msgid ""
 "  }\n"
 "\\end{table}"
 msgstr ""
+"\\begin{table}[H]\n"
+"  \\hommtable{26}{\n"
+"    \\centering\n"
+"    \\medskip\n"
+"    \\textbf{Punkty Zwycięstwa na koniec gry}\\\\\n"
+"    \\bigskip\n"
+"    \\begin{tabularx}{0.95\\linewidth}{>{\\hsize=0.12\\hsize\\linewidth=\\hsize}X\n"
+"        >{\\hsize=0.8\\hsize\\linewidth=\\hsize}X\n"
+"      }\n"
+"      \\darkcell[0.8]{PZ} & \\darkcell[0.8]{Kategoria} \\\\\n"
+"      \\darkcell[1.2]{1} &\n"
+"      \\lightcell[1.2]{za każdą kontrolowaną Osadę lub Kopalnię} \\\\\n"
+"      \\darkcell[1.2]{1} &\n"
+"      \\lightcell[1.2]{za każdy budynek w Twoim Mieście} \\\\\n"
+"      \\darkcell[1.2]{1} &\n"
+"      \\lightcell[1.2]{za każde dwa Artefakty w \\mbox{twojej} Talii (razem z usuniętymi)} \\\\\n"
```

Comment:
Rozbij to proszę na trzy wiersze, np. tak:
za każde dwa Artefakty
w\~twojej Talii (razem
z~usuniętymi)

Być może u góry "  \\hommtable{26}{\n" trzeba będzie zmienić na 27. Poza tym wszystko wygląda OK.

Reply:
<img width="850" height="170" alt="Image" src="https://github.com/user-attachments/assets/727650fb-0f1a-4546-a15c-0db6d607833e" />
Chodzi o to, że litery przyklejone do krawędzi czegokolwiek wyglądają, jakby coś było popsute.